helm/README.md

```
@@ -0,0 +1,121 @@
+# MCP Mesh Helm Charts
+
+Kubernetes deployment charts for MCP Mesh - a distributed agent orchestration framework.
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)